src/LDLFactorizations.jl

```diff
@@ -438,7 +438,7 @@ The components of the factorization can be accessed via `getproperty`:
 In order to avoid zero pivots during the factorization, the user can regularize the matrix by modifying 
 `LDL.r1` for the `LDL.n_d` first pivots and `LDL.r2` for the other pivots with tolerance `LDL.tol`.
 """
-mutable struct LDLFactorization{T <: Real, Ti <: Integer, Tn <: Integer, Tp <: Integer}
+mutable struct LDLFactorization{T <: Number, Ti <: Integer, Tn <: Integer, Tp <: Integer}
```

Comment:
Hoping to get rid of `T <: Number` and just leave it as `T`.

Reply:
Yes. Actually, I realized this morning that `AbstractInterval <: Real`, so it should work with this PR out of the box 🤞 .